guide/getting-started/software.md

```diff
@@ -115,47 +121,41 @@ Payment processing applications offer easy-to-use online stores and point-of-sal
 
 </div>
 
-## Exchanges
+## Nodes
 
 <div class="center" markdown="1">
 
-Exchanges lets users swap between currencies and typically fall in three general categories. Consumer-focused exchanges let users make trades directly with the exchange. This is ideal for occasional use.
-
-
 {% include image.html
-   image = "/assets/images/guide/getting-started/software/exchange-example.jpg"
-   retina = "/assets/images/guide/getting-started/software/exchange-example@2x.jpg"
-   alt-text = "Illustrative interface for exchanging bitcoin"
-   caption = 'Exchanges allow for trading between different currencies. Texture by [Bilal O.](https://unsplash.com/@lightcircle){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
+   image = "/assets/images/guide/getting-started/software/node-ui-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/node-ui-example@2x.jpg"
+   alt-text = "Illustrative interface for node software"
+   caption = 'Node applications connect to and participate in the bitcoin network. Texture by [okeykat](https://unsplash.com/@okeykat){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
    width = 400
-   height = 388
+   height = 426
    layout = "float-right-desktop"
 %}
 
-Exchanges focused on investors and traders usually list many different cryptocurrencies. They allow users to trade with each other by submitting buy and sell requests on currency pairs. These products are generally for experienced users. They offer complex trading features and countless ways to analyze price movements.
+Bitcoin [node]({{ '/guide/glossary/node/' | relative_url }}) software connects to and participates in the Bitcoin network. [Full nodes]({{ '/guide/glossary/node/#full-nodes/' | relative_url }}) verify transaction data on the blockchain and ensure the Bitcoin rules are being followed by network participants. A [light node]({{ '/guide/glossary/node/#light-nodes/' | relative_url }}) connects to the Bitcoin network but does not verify or enforce the rules. Some wallet software comes with built-in full node capabilities, such as [Bitcoin Core]({{ '/guide/glossary/#bitcoin-core-client/' | relative_url }}), but most wallet software act as light nodes. 
```

Comment:
```suggestion
Bitcoin [node]({{ '/guide/glossary/node/' | relative_url }}) software connects to and participates in the Bitcoin network. Nodes typically download and broadcast user transactions, and optionally help verify blockchain data more broadly. Some wallet software comes with built-in node capabilities, such as [Bitcoin Core]({{ '/guide/glossary/#bitcoin-core-client/' | relative_url }}), but most wallet software connects to external nodes. Learn more on the [Node page]({{ '/guide/glossary/node/' | relative_url }}) in the glossary.
```
I find the full/light node info too detailed for this page. It's meant to provide a light overview and jumping-off points deep down into the endless depths the guide.

I tried to replace it with a simple statement about what a node does (download and broadcast transactions), with the addition that it can optionally do verification (and other tuff). Let's deep-link into the nodes page for the details.

Reply:
Addressing in fresh commit as this has got outdated.